cypress/page_object/task3/Footer.js

```diff
@@ -0,0 +1,16 @@
+class Footer {
+
+    // iconClass: '.em-facebook' , '.em-youtube' , '.em-instagram'
+    getSocialMediaLinkByClass(iconClass) {
+        cy.scrollTo('bottom');
+        cy.get('div[class="footer-copy footer-inverse"]')
+            .find(iconClass).parent().should('have.attr', 'target');
+        return cy.get('div[class="footer-copy footer-inverse"]')
```

Comment:
the code is duplicated here: cy.get('div[class="footer-copy footer-inverse"]').find(iconClass).parent()